### ISSUE TITLE
log_debug business... can we agree most of this is useless?

### ISSUE THREAD

**Issue by mick-p1982** [zturtleman/mm3d#64]
Is "log_debug" for bug reports? I think it's a lot to maintain for no real value.

It's probably only active in debug builds... meaning it's not handy for regular users if so. I can't see what it's for myself.

EDITED: I can see how it's useful in development, but not in public code.

**Comment by zturtleman** [comment]
Log messages are enabled with `mm3d.exe --debug --warnings --errors`. log_debug() has a lot of useless spam from working on specific issues though.

**Comment by mick-p1982** [comment]
So, I think instances of it should be removed... on the basis of clutter makes code hard to read. It should be a macro that emits no code in release builds.

What about LOG_PROFILE? I see this, it's not used consistently, so I've removed it from Tools code where I've come across it. I haven't looked into it; what it does, etc.

I've lately been going over most of the tools files, since I've changed some things about how they work... mainly how the UI elements work. It's slow work, like I can only do a handful in a day. Even doing simple programming work, it takes hours. I think why we don't have lots of good noncommercial software is it just takes so long to write.
